### PR TITLE
Refactor layouts to make markup compliant with Layout Builder

### PR DESCRIPTION
**Pull request by azinck**
I haven't finished this work because I wanted you to look at it, @dcmouyard.  The problem I'm trying to address is the fact that our layout markup doesn't currently allow users to drag and drop blocks within the layout.  That functionality is just totally broken.  That's because we're stripping out all the classes that Layout Builder adds to our regions.  This PR fixes that for 3 of our layouts.  I'll do the rest if you find these changes acceptable.  I tried to keep the changes to a minimum, but we do have to have some sort of wrapper around each region.

### CODE REVIEW

File: services/drupal/web/modules/custom/epa_layouts/layouts/epa-four-column.html.twig [hunk 1]
```diff
@@ -17,28 +17,28 @@
...
-{% endif %}
+  {%
+    set classes = [
+      'l-grid',
+      'l-grid--' ~ columns  ~ '-col'
+  ]
+  %}
+{%  endif %}
+
+<div{{ attributes.addClass(classes) }}>
```
Comment (azinck) [inline services/drupal/web/modules/custom/epa_layouts/layouts/epa-four-column.html.twig:28]: We need to not output this wrapper div if there's only a single column.

[the remark this answers is not in the document]
Comment (dcmouyard) [reply services/drupal/web/modules/custom/epa_layouts/layouts/epa-four-column.html.twig:28]: I would prefer to keep the old logic of only outputting wrapper markup if there's content in more than one region. I _think_ that will still be compatible with Layout Builder.

**Comment by azinck** [comment]
Ok, @dcmouyard -- this is ready for your review.  I removed some code duplication by making a single template file for our 2, 3, and 4-column layouts.  Theoretically this could be done for our 1-column layout as well, but it has different region names and a slightly different structure so I decided not to mess with it.  Let me know what you think.

**Review by dcmouyard — APPROVED** [review]
Looks good to me! 👍

**Pull request merged**